newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js

```diff
@@ -82,6 +85,9 @@ export default class PlatformSpecificAssetsDialog extends React.Component<
       androidIconResourceNames: androidSizes.map(size =>
         project.getPlatformSpecificAssets().get('android', `icon-${size}`)
       ),
+      androidWindowSplashScreenAnimatedIconResourceName: project
+        .getPlatformSpecificAssets()
```

Comment:
can the `project.getPlatformSpecificAssets()` be factorised at the start of the function?

Reply:
Fixed